services/bower/bower-license.tester.js

```diff
@@ -6,15 +6,6 @@ t.create('licence')
   .get('/bootstrap.json')
   .expectBadge({ label: 'license', message: 'MIT' })
 
-t.create('license not declared')
-  .get('/bootstrap.json')
-  .intercept(nock =>
-    nock('https://libraries.io')
-      .get('/api/bower/bootstrap')
-      .reply(200, { normalized_licenses: [] })
-  )
-  .expectBadge({ label: 'license', message: 'missing' })
-
```

Comment:
Our Bower badges had a few such mocked requests. With the hard requirement that we have the API Keys enabled in CI (we **always** exceed the rate limit with our ~20+ tests without auth), trying to mock these becomes increasingly difficult.

We've used some nock tricks elsewhere (e.g. `skipWhen`) that I considered, but I actually found these mocked tests were either duplicative/unnecessary tests of a render helper (as was the case here) and/or the test scenario would be better covered with unit tests.